trailmet/algorithms/prune/network_slimming_prune.py

```diff
@@ -0,0 +1,232 @@
+import os
```

Comment:
there should only be a single file netwoek_slimming.py and it should have everything in a modular manner organized into classes.